README.mdown

```diff
@@ -1,131 +1,211 @@
   - "TITLESHORT": abbreviated journal title
   - "PARENT": independent parent style
-  - "DOCUMENTATION": documentation URL (ampersands in URLs are automatically escaped)
-  - "FORMAT": citation format (e.g. "numeric")
-  - "FIELD": field (e.g. "medicine")
+  - "DOCUMENTATION": documentation URL
+  - "FORMAT": citation format (e.g., "numeric")
+  - "FIELD": field (e.g., "medicine")
   - "ISSN": print ISSN
   - "EISSN": electronic ISSN
-  - "LOCALE": locale code for [localization of citations and bibliographies](https://github.com/citation-style-language/locales/wiki)
-- capitalization of column headers is not important. The script uses a case-insensitive match, so "TITLE" is the same as "title" or "Title".
-- there is no upper limit on the number of metadata fields.
-- some journals may not have a value for each metadata field. For example, online-only journals typically don't have a print ISSN. In this case, just leave the field empty, or fill it with a single character, such as "x". 
-
-#### \_template.csl
+  - "LOCALE": locale code for style localization (e.g., "en-US" for US English; for a list of common locale codes, see the [CSL locales repository](https://github.com/citation-style-language/locales/wiki))
+- ampersands are automatically escaped for the "TITLE", "TITLESHORT", and "DOCUMENTATION" fields.
+- not every journal will have a value for each metadata field.
+  For example, online-only journals won't have a print ISSN.
+  In this case, just leave the field empty, or fill it with a single character, such as "x".
+
+##### \_template.csl
+`_template.csl` is the template for the dependent CSL styles.
+An example:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="#LOCALE#">
+  <!-- #XML-COMMENT# -->
+  <info>
+    <title>#TITLE#</title>
+    <id>http://www.zotero.org/styles/#IDENTIFIER#</id>
+    <link href="http://www.zotero.org/styles/#IDENTIFIER#" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-society-for-microbiology" rel="independent-parent"/>
+    <link href="#DOCUMENTATION#" rel="documentation"/>
+    <category citation-format="numeric"/>
+    <category field="biology"/>
+    <issn>#ISSN#</issn>
+    <eissn>#EISSN#</eissn>
+    <updated>2015-05-08T17:36:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+</style>
+```
 
-`_template.csl` must contain the XML template for the dependent CSL styles. An example is shown below:
+This template file should include all metadata that is the same for all journals.
+In contrast, template placeholders should be used for metadata that differs between journals.
+These placeholders, in all-caps and enclosed in pound signs (e.g., `#TITLE#`), are replaced by journal metadata during style generation.
```

Comment:
The "pound sign" is for me £ and I would rather call this "hash". But I see that this depends on the region we are living in, see https://en.wikipedia.org/wiki/Number_sign